generated/typespec-ts/src/api/operations.ts

```diff
@@ -0,0 +1,1708 @@
+        top_p: body["topP"],
+        n: body["n"],
+        max_tokens: body["maxTokens"],
+        stop: body["stop"],
+        presence_penalty: body["presencePenalty"],
+        frequency_penalty: body["frequencyPenalty"],
+        logit_bias: body["logitBias"],
+        user: body["user"],
+        stream: body["stream"],
+      },
+    });
+}
+
+export async function _createChatCompletionDeserialize(
+  result: CreateChatCompletion200Response | CreateChatCompletionDefaultResponse
+): Promise<CreateChatCompletionResponse> {
+  if (isUnexpected(result)) {
+    throw result.body;
+  }
+
+  return {
+    id: result.body["id"],
+    object: result.body["object"],
+    created: new Date(result.body["created"]),
+    model: result.body["model"],
+    choices: (result.body["choices"] ?? []).map((p) => ({
+      index: p["index"],
+      message: {
+        role: p.message["role"] as any,
+        content: p.message["content"],
+        functionCall: !p.message.function_call
+          ? undefined
+          : {
+              name: p.message.function_call?.["name"],
+              arguments: p.message.function_call?.["arguments"],
+            },
+      },
+      finishReason: p["finish_reason"] as any,
+    })),
+    usage: !result.body.usage
+      ? undefined
+      : {
+          promptTokens: result.body.usage?.["prompt_tokens"],
+          completionTokens: result.body.usage?.["completion_tokens"],
+          totalTokens: result.body.usage?.["total_tokens"],
+        },
+  };
+}
+
+export async function createChatCompletion(
+  context: Client,
+  body: CreateChatCompletionRequest,
+  options: CreateChatCompletionOptions = { requestOptions: {} }
+): Promise<CreateChatCompletionResponse> {
+  const result = await _createChatCompletionSend(context, body, options);
+  return _createChatCompletionDeserialize(result);
+}
+
+export function _createFineTuningJobSend(
+  context: Client,
+  job: CreateFineTuningJobRequest,
+  options: CreateFineTuningJobOptions = { requestOptions: {} }
+): StreamableMethod<
+  CreateFineTuningJob200Response | CreateFineTuningJobDefaultResponse
+> {
+  return context
+    .path("/fine_tuning/jobs")
+    .post({
+      ...operationOptionsToRequestParameters(options),
+      body: {
+        training_file: job["trainingFile"],
+        validation_file: job["validationFile"],
+        model: job["model"],
+        hyperparameters: !job.hyperparameters
+          ? undefined
+          : { n_epochs: job.hyperparameters?.["nEpochs"] },
+        suffix: job["suffix"],
+      },
+    });
+}
+
+export async function _createFineTuningJobDeserialize(
+  result: CreateFineTuningJob200Response | CreateFineTuningJobDefaultResponse
+): Promise<FineTuningJob> {
+  if (isUnexpected(result)) {
+    throw result.body;
+  }
+
+  return {
+    id: result.body["id"],
+    object: result.body["object"],
+    createdAt: new Date(result.body["created_at"]),
+    finishedAt: new Date(result.body["finished_at"]),
```

Comment:
[Compile Error]

src/api/operations.ts:345:26 - error TS2769: No overload matches this call.
  Overload 1 of 4, '(value: string | number | Date): Date', gave the following error.
    Argument of type 'string | null' is not assignable to parameter of type 'string | number | Date'.
      Type 'null' is not assignable to type 'string | number | Date'.
  Overload 2 of 4, '(value: string | number): Date', gave the following error.
    Argument of type 'string | null' is not assignable to parameter of type 'string | number'.
      Type 'null' is not assignable to type 'string | number'.

345     finishedAt: new Date(result.body["finished_at"]),